docs/reference/module-options.md

```diff
@@ -372,3 +372,442 @@ module.exports = {
+        label: 'Home'
+      }
+    ]
+  },
+  // ...
+}
+```
+
+## Options for page type modules
+
+Option settings in this section apply to all page types (modules that extend `@apostrophecms/page-type`).
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`scene`](#scene) | String | Change the "scene" for this page type from the default `'public'`. |
+
+### `scene`
+
+Scenes are contexts in which certain sets of front end assets are delivered. Normally, anonymous site visitors receive only the stylesheets and scripts included in the `'public'` asset scene (those that are placed in the module's `ui/public` directory). If your page will use assets, such as Apostrophe's modals, that are normally reserved for logged-in users you can set `scene: 'apos'` in order to load them with pages of this type.
+
+#### Example
+
+```javascript
+// modules/fancy-form-page/index.js
+module.exports = {
+  extend: '@apostrophecms/page-type',
+  options: {
+    scene: 'apos'
+  },
+  // ...
+}
+```
+
+## Options for piece page types
+
+Option settings in this section apply to all piece page types (modules that extend `@apostrophecms/piece-page-type`).
+
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`perPage`](#perpage) | Integer | Set the number of pieces to be |
+| [`next`](#next) | Boolean/Object | Enable and optionally configure the `req.data.next` object. |
+| [`piecesFilters`](#piecesfilters) | Array | Configure pieces filters for index pages. |
+| [`piecesModuleName`](#piecesmodulename) | String | Specify the associated piece type. |
+| [`previous`](#previous) | Boolean/Object | Enable and optionally configure the `req.data.previous` object. |
+
+### `perPage`
+
+For piece pages, the `perPage` option, expressed as an integer, defines the number of pieces that will be added to the `req.data.pieces` array when the page is served. The specific pieces in the set will be based on the total number of pieces and the `page` query parameter in the request.
+
+This is, more simply, the number of pieces that will normally be displayed on each page of a [paginated index page](/reference/glossary.md#index-page). This value defaults to 10.
+<!-- TODO: Change index page link to a guide on index pages when available. -->
+
+#### Example
+
+```javascript
+// modules/article-page/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    perPage: 12
+  },
+  // ...
+}
+```
+
+### `next`
+
+If set to `true`, Apostrophe will include the next piece, based on the [sort option](#sort), on `req.data.next` when serving a [show page](/reference/glossary.md#show-page). This is useful to add links to a show page directing visitors to the next item in a series (e.g., the next oldest blog post). If not set, `req.data.next` will not be available.
+
+`next` can also be set to an object, which will be used as a query builder for retrieving that next piece document.
```

Comment:
... Yeah. Let's let this language stand. Sure, a "query builder" can refer to such an object containing query builder method names and values to pass to them. We can use the phrase "query builder method" to refer to the individual methods when there is ambiguity.